### PR TITLE
Docs: feature flag refresh

### PR DESCRIPTION
**Pull request by ramonjd**
## What?
Updating language in feature flags doc because I can't help myself.

## Why?
I was reading it and thought it needed a little spring cleaning. 

## How?
⌨️ 

## Testing Instructions
Are there any typos? Can you understand the meaning?

### CODE REVIEW

File: docs/how-to-guides/feature-flags.md [hunk 1]
```diff
@@ -33,68 +37,68 @@ if ( process.env.IS_GUTENBERG_PLUGIN ) {
...
-`undefined` evaluates to `false` so the plugin only feature will not be executed within core.
+`undefined` evaluates to `false` so the plugin-only feature will not be executed.
 
 ### Dead Code Elimination
 
-When building code for production, webpack 'minifies' code (https://en.wikipedia.org/wiki/Minification_(programming)), removing the amount of unnecessary JavaScript as much as possible. One of the steps involves something known as 'dead code elimination'.
+For production builds, webpack ['minifies'](https://en.wikipedia.org/wiki/Minification_(programming)) the code, removing as much unnecessary JavaScript as it can. 
 
-When the following code is encountered, webpack determines that the surrounding `if`statement is unnecessary:
+One of the steps involves something known as 'dead code elimination'. For example, when the following code is encountered, webpack determines that the surrounding `if`statement is unnecessary:
```
Comment (aaronrobertshaw) [inline docs/how-to-guides/feature-flags.md]: Nit: Maybe a space here is good?
```suggestion
One of the steps involves something known as 'dead code elimination'. For example, when the following code is encountered, webpack determines that the surrounding `if` statement is unnecessary:
```

**Review by aaronrobertshaw — APPROVED** [review]
Thanks for improving the docs @ramonjd 👍 

This reads well to me.

File: docs/how-to-guides/feature-flags.md (hunk 1 above)
Comment (ramonjd) [reply docs/how-to-guides/feature-flags.md]: Good spotting. Thanks!

**Review by gziolo — APPROVED** [review]
Thank you for all improvements 👍

**Pull request merged**